src/Rules/Resources/HowToFix.resx

```diff
@@ -122,12 +122,15 @@
     <comment>The description of a problem with an element currently under inspection. The term "BoundingRectangle" should not be translated (i.e. it should be treated as a loanword).</comment>
   </data>
   <data name="ButtonWithSplitButtonParentPattern" xml:space="preserve">
-    <value>A button element should only support one of Invoke, Toggle, or ExpandCollapse patterns when a splitbutton is the parent. </value>
-    <comment>The description of a problem with an element currently under inspection. The terms "Invoke", "Toggle", and "ExpandCollapse" should not be translated (i.e. they should be treated as loanwords).</comment>
+    <value>Modify the button to support exactly one of the following patterns:
```

Comment:
the original value provided relevant information as to why the rule was failing. (e.g. A button element should only support one of Invoke, Toggle, or ExpandCollapse patterns when a splitbutton is the parent. ) Love the new how to fix information but wondering if we should keep the _why it failed_ information as well. Also, these new instructions are longer than the previous ones, how does this impact the UI in Accessibility Insights for Windows?

Reply:
> wondering if we should keep the why it failed information as well.

Agree on this point!

> These new instructions are longer than the previous ones, how does this impact the UI in Accessibility Insights for Windows?

I don't think we need to worry about that, considering some of our existing how to fix strings are already quite long (for example, https://github.com/microsoft/axe-windows/blob/main/src/Rules/Resources/HowToFix.resx#L251) and they show up well in AI for windows:

[Screenshot of long how to fix string rendered in AIWin]
<img width="311" alt="HowToFixScreenshot" src="https://user-images.githubusercontent.com/16010855/198382372-e99f178d-a289-47df-be6b-e6392f7cdcab.png">

Reply:
> wondering if we should keep the why it failed information as well.

Actually, I think @codeofdusk might have mentioned that part of the motivation to replace these strings was that they are duplicated, and it does look like the original strings here do also exist in the rules descriptions. @codeofdusk please confirm, but if that is the case then I don't think we need to keep the why it failed information in these strings as well.

Reply:
Correct – these strings were duplicated from the rule descriptions.

Reply:
got it! will the user still have access to the strings in all views (automated checks and inspect tree)?

Reply:
@ferBonnin, here's the zoomed out picture of my screenshot above. You can see that the how to fix strings (the ones being changed in this PR) are at the bottom in the 'fix the following' section and the rule descriptions (which describe why the failure occurred) are still above and visible in the 'test results' section.

[Screenshot of full how to fix page in AIWin]
<img width="644" alt="LargerHowToFix" src="https://user-images.githubusercontent.com/16010855/198401096-eaeb84e6-6243-4258-8b56-c966cd8e9485.png">